myVK/myVK/Services/NetworkService.swift

```diff
@@ -3,6 +3,8 @@
 
 import Alamofire
 import Foundation
+import RealmSwift
+import SwiftyJSON
```

Comment:
```suggestion
import SwiftyJSON
```
Неиспользуемый импорт